08-react-2/classworks/Pogudo Tanya/src/pages/home/Communities.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import "./style.css";
+import Student from "./react_img.png";
+import { Link } from "react-router-dom";
+import { useRouteMatch } from "react-router-dom";
+
+export default function Communities(props) {
+  const [value, setValue] = useState(props.func);
+
+  return (
+    <div className="block_list">
+      <div className="img_size">
+        <img
+          alt=" "
+          className="img_size"
+          src={props.post.header_img ? props.post.header_img : Student}
+        />
+      </div>
+      <div className="button_div">
+        <button
+          className="button_click_title"
+          onClick={() => {
+            props.func(props.post.display_name,props.post.display_name_prefixed, props.post.header_img,props.post.display_name);
+          }}
+        >
+          <Link to={`/home/${props.post.display_name}`} className="title_sub">
+            {props.post.display_name_prefixed}
+          </Link>
+        </button>
+
+        <div className="members_style">
+          {props.post.subscribers === null
```

Comment:
usually we don't make multiple ternary operators in a row as it's hard to read and understand. please make a function to format number and call it here